File: src/main/java/client/Sound.java
package client;

import java.io.*;

public class Sound {
    private File audioFile;
    private String name;

    public Sound(String name) {
        try {
            this.audioFile = new File(this.getClass().getClassLoader().getResource("sounds/"+name+".wav").getFile());
        } catch (NullPointerException e) {
            e.printStackTrace();
        }
        this.name = name;
    }

    public void play(){
    }

    public String getName() {
        return name;
    }
}

File: src/main/java/client/Sound.kt
package client

import java.io.*

class Sound(val name: String) {
    private var audioFile: File? = null

    init {

        this.javaClass.classLoader.getResource("sounds/$name.wav")?.let {
            this.audioFile = File(it.file)
        } // TODO: Logger warn here if null.
    }

    fun play() {}
}
